string object_name
---
geometry_msgs/Pose pose
bool success
string message
